eng/common/scripts/get-markdown-files-from-changed-files.ps1

```diff
@@ -0,0 +1,47 @@
+param (
+    # The root repo we scaned with.
+    [string[]] $RootRepo = "./",
+    # The target branch to compare with.
+    [string] $targetBranch = ${env:SYSTEM_PULLREQUEST_TARGETBRANCH}
```

Comment:
```suggestion
    [string] $targetBranch = "origin/${env:SYSTEM_PULLREQUEST_TARGETBRANCH}"
```

This should include the full branch name instead of assuming everything is from origin in the script below.